src/routes/OverViewPage/OverViewPageHeader.js

```diff
@@ -1,44 +1,87 @@
 import React from 'react';
-import { Flex, FlexItem } from '@patternfly/react-core';
-import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+import {
+  Button,
+  Flex,
+  FlexItem,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+import {
+  ExternalLinkAltIcon,
+  OpenDrawerRightIcon,
+} from '@patternfly/react-icons';
 import {
   PageHeader,
   PageHeaderTitle,
 } from '@redhat-cloud-services/frontend-components/PageHeader';
 import InsightsLink from '@redhat-cloud-services/frontend-components/InsightsLink';
 import { RemediationsPopover } from '../RemediationsPopover';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+import PropTypes from 'prop-types';
+
+export const OverViewPageHeader = ({ hasRemediations }) => {
+  const { quickStarts } = useChrome();
 
-export const OverViewPageHeader = () => (
-  <PageHeader>
-    <PageHeaderTitle
-      className="pf-v5-u-mb-lg"
-      title={
-        <Flex
-          direction={{ default: 'row' }}
-          spaceItems={{ default: 'spaceItemsSm' }}
-          alignItems={{ default: 'alignItemsCenter' }}
-        >
-          <FlexItem>Remediation plans</FlexItem>
+  return (
+    <PageHeader className="pf-v5-u-pb-lg">
+      <Flex
+        justifyContent={{ default: 'spaceBetween' }}
+        alignItems={{ default: 'alignItemsFlexStart' }}
+      >
+        <FlexItem flex={{ default: 'flex_1' }}>
+          <Stack hasGutter>
+            <StackItem>
+              <PageHeaderTitle
+                title={
+                  <Flex
+                    spaceItems={{ default: 'spaceItemsSm' }}
+                    alignItems={{ default: 'alignItemsCenter' }}
+                  >
+                    <FlexItem>Remediation plans</FlexItem>
+                    <FlexItem>
+                      <RemediationsPopover />
+                    </FlexItem>
+                  </Flex>
+                }
+              />
+            </StackItem>
+
+            <StackItem>
+              <p>
+                Remediation plans use Ansible playbooks to resolve issues
+                identified by Red Hat Insights.
+                <InsightsLink
+                  className="pf-v5-u-ml-md"
+                  to="https://docs.redhat.com/en/documentation/red_hat_insights/1-latest/html/red_hat_insights_remediations_guide/index"
+                  style={{ textDecoration: 'none' }}
+                >
+                  Learn more <ExternalLinkAltIcon />
```

Comment:
We need to remove this learn more thing as we replaced it with quick start launch. Mocks don't have `Learn more`

Reply:
I have a meeting today to discuss this, in some mocks its there and others its removed. I will add the result to this comment and then merge/make changes then merge

Reply:
This is removed now!